libtsuba/src/RDGPartHeader.cpp

```diff
@@ -213,17 +228,17 @@ RDGPartHeader::Validate() const {
           "edge_property path doesn't contain a slash (/): {}", md.path());
     }
   }
-  if (topology_path_.empty()) {
-    return KATANA_ERROR(ErrorCode::InvalidArgument, "topology_path is empty");
-  }
-  if (topology_path_.find('/') != std::string::npos) {
-    return KATANA_ERROR(
-        ErrorCode::InvalidArgument,
-        "topology_path doesn't contain a slash (/): {}", topology_path_);
+
+  auto res = topology_metadata_.Validate();
+  if (!res) {
+    return res;
```

Comment:
If you like, this could just be `KATANA_CHECKED(topology_metadata_.Validate());`.